pxtwinrt/hiddeploy.ts

```diff
@@ -77,12 +78,15 @@ namespace pxt.winrt {
                     pxt.debug(`hid enumerate ${devices.length} devices`);
                     const device = devices[0];
                     pxt.debug(`hid connect to ${device.name} (${device.id})`);
+                    deviceId = device.id;
                     return whid.fromIdAsync(device.id, Windows.Storage.FileAccessMode.readWrite);
                 })
                 .then((r: WHID) => {
                     this.dev = r;
                     if (!this.dev) {
                         pxt.debug("can't connect to hid device");
+                        let status = Windows.Devices.Enumeration.DeviceAccessInformation.createFromId(deviceId).currentStatus;
+                        pxt.reportError("winrt_device", `could not connect to HID device; device status: ${status}`);
```

Comment:
Is this shown to the user? Use lf macro

Reply:
This is just for us to maybe help diagnose problems, users are shown a more generic and user friendly error message (see in the `.catch` at the end of this chain)